### PR TITLE
Lint/style/readability fixes.

### PR DESCRIPTION
**Pull request by nimf**

### CODE REVIEW

**Review by Jennnnny — APPROVED** [review]

File: grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java [hunk 1]
```diff
@@ -70,8 +71,7 @@
   private int maxSize = DEFAULT_MAX_CHANNEL;
   private int maxConcurrentStreamsLowWatermark = DEFAULT_MAX_STREAM;
 
-  @VisibleForTesting
-  final Map<String, AffinityConfig> methodToAffinity = new HashMap<String, AffinityConfig>();
+  @VisibleForTesting final Map<String, AffinityConfig> methodToAffinity = new HashMap<>();
```
Comment (vnorigoog) [inline grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java:74]: readability nit: this line has "@VisibilityForTesting" on the same line but the next line has it on its own line. I think you should stick one of the 2 styles and be consistent..

File: grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java [hunk 2]
```diff
@@ -878,10 +878,10 @@ public String authority() {
       MethodDescriptor<ReqT, RespT> methodDescriptor, CallOptions callOptions) {
```
Comment (vnorigoog) [inline grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java:878]: nit: why does this line have 2 extra spaces at the start?

File: grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java [hunk 3]
```diff
@@ -948,16 +948,22 @@ public ConnectivityState getState(boolean requestConnection) {
...
+          break;
+        case TRANSIENT_FAILURE:
+          transientFailure++;
+          break;
+        case CONNECTING:
+          connecting++;
+          break;
+        case IDLE:
+          idle++;
+          break;
```
Comment (vnorigoog) [inline grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java:966]: is "default" option for switch needed?

**Review by vnorigoog — APPROVED** [review]
good stuff overall..
some nit comments added.
otherwise LGTM

File: grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java (hunk 1 above)
Comment (nimf) [reply grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java:74]: Yeah, I know, but this is how `gradle googleJavaFormat` works.

File: grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java (hunk 2 above)
Comment (nimf) [reply grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java:878]: I guess `googleJavaFormat` makes a 4-space indentation for a multiline method signature.

**Pull request merged**